alternative-log-storage/aws/dynamodb/main.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
```

Comment:
you should not add provider here - they need to use it as module or fork it

Reply:
a2f22176e9aa204ce0eba5a9b77e5a7c4e38bc60